docs/guides/integration/github.md

```diff
@@ -231,7 +231,7 @@ Or invalidate it when the lockfile changes:
 
 ```yaml title="example.yml"
 - name: Define a cache dependency glob
-  uses: astral-sh/setup-uv@v5
+  uses: astral-sh/setup-uv@v6
   with:
     enable-cache: true
     cache-dependency-glob: "uv.lock"
```

Comment:
CC @eifinger is this still a good example?

Reply:
Mh while this still works and might be a valid usecase in an exotic usecase (make sure ONLY the lock file invalidates the cache) I think it would be better if we remove examples involving `cache-dependency-glob`.

The new default should be sufficient for most users and I don't think examples should contain edge cases most users will never need to think about.

For those edge cases the README in `setup-uv` still contains the information

Reply:
Sure, I will put it on my todo list 👍